File: Lab5.py
array = [123, 125.4, 123.00246, 125.4003, 125, 127.27]
n = len(array)
num = float(input("Enter num :"))
max = min = array[0]
indexMax = indexMin = sum = 0
variance = 0
for i in range(n):
    if array[i] > max:
        max = array[i]
        indexMax = i
    elif array[i] < min:
        min = array[i]
        indexMin = i
    if array[i] == num:
        print("Your index :", i)
    sum += array[i]
print(f"max : {max:.2f} at index : {indexMax}")
print(f"min : {min:.2f} at index : {indexMin}")
print("average :", "{:.2f}".format(sum/n))

for i in range(n):
    variance += (array[i] - sum/n) * (array[i] - sum/n)
variance /= n
print("variance :", "{:.2f}".format(variance))

File: Lab5.java
import java.util.Scanner;

public class Lab5 {
    public static void main(String[] args) {
        float[] array = {123.456f, 45.6789f, 7.89123f};
        int n = array.length;
        float max = array[0], min = array[0], sum = 0;
        int indexMax = 0, indexMin = 0;
        float variance = 0;
        try (Scanner x = new Scanner(System.in)) {//try
        System.out.print("Enter num :");
        float num = x.nextFloat();

        for(int i = 0; i < n; i++) {
            if (array[i] > max) {
                max = array[i];
                indexMax = i;
            }
            if (array[i] < min) {
                min = array[i];
                indexMin = i;
            }
            if (array[i] == num) {
                System.out.println("Your index : " + i);
            }
            sum += array[i];
        }
        }//cau use x.close(); instead of try
        System.out.printf("max : %.2f at index : %d%n", max, indexMax);
        System.out.println("min : " + String.format("%.2f", min) + " at index : " + indexMin);
        System.out.println("average : " + String.format("%.2f", sum/n));

        for(int i = 0; i < n; i++)
            variance += (array[i] - sum/n) * (array[i] - sum/n);
        variance /= n;
        System.out.println("variance : " + String.format("%.2f", variance));
    }
}